src/pages/demo/emailContacts/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { v4 as uuidv4 } from 'uuid'
+import { drupalClient } from '@/utils/drupalClient'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+import { DrupalNode } from 'next-drupal'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+import EmailContact from '@/components/paragraph/email_contact'
+
+interface EmailContactsPageProps {
+  emailContacts: DrupalNode[]
+}
+
+const EmailContactsPage = ({ emailContacts }: EmailContactsPageProps) => {
+  if (!emailContacts) emailContacts = []
+
+  return (
+    <>
+      <Container className="container">
+        {emailContacts.map((paragraph) => (
+          <EmailContact key={uuidv4()} paragraph={paragraph} />
```

Comment:
`paragraph.id` or `node.id` is always present and is a uuid. If there aren't problems with using that as the key, we could just use that and not use the library.